en: update dm-worker config doc for 2.0 (#34)
pingcap/docs-dm: en/dm-worker-configuration-file.md

# DM-worker Configuration File

> This document introduces the configuration of DM worker, including a configuration file template and instructions about what the configuration parameters mean.

Reviewer: ```suggestion
This document introduces the configuration of DM worker, including a configuration file template and a description of each configuration parameter in this file.
```

## Configuration file template

The following is a configuration file template of the DM-worker:

```toml
# Worker Configuration.
name = "worker1"

# Log configuration.
log-level = "info"
log-file = "dm-worker.log"

# DM-worker listen address.
worker-addr = ":8262"
advertise-addr = "127.0.0.1:8262"
join = "127.0.0.1:8261,127.0.0.1:8361,127.0.0.1:8461"
```

## Configuration parameters

### Global

| Parameter        | Description                           |
| :------------ | :--------------------------------------- |

> | `name` | Identifies a DM-worker. |

Reviewer: ```suggestion
| `name` | Marks a DM-worker name. |
```

> | `log-level` | Specifies a log level from `debug`, `info`, `warn`, `error`, and `fatal`. The default log level is `info`. |

Reviewer: ```suggestion
| `log-level` | Specifies the log level. The optional values are `debug`, `info`, `warn`, `error`, and `fatal`. The default log level is `info`. |
```

> | `log-file` | Specifies the log file directory. If not specified, the logs are printed onto the standard output. |

Reviewer: ```suggestion
| `log-file` | Specifies the log file directory. If this parameter is not specified, the logs are printed onto the standard output. |
```

| `worker-addr` | Specifies the address of DM-worker which provides services. You can omit the IP address and specify the port number only, such as ":8262". |

> | `advertise-addr` | Specifies the address that DM-worker advertises to the outside world. |

Reviewer: ```suggestion
| `advertise-addr` | Specifies the address that DM-worker is advertised externally. |
```
